Examples/GithubPaginatedSearch.swift

```diff
@@ -93,36 +93,40 @@ class GithubPaginatedSearchViewController: UIViewController {
     @IBOutlet weak var searchResults: UITableView?
     @IBOutlet weak var status: UILabel?
     @IBOutlet weak var loadNextPage: UILabel?
-    
+
     private let disposeBag = DisposeBag()
-    
+
     override func viewDidLoad() {
         super.viewDidLoad()
-        
+
         let searchResults = self.searchResults!
-        
-        searchResults.register(UITableViewCell.self, forCellReuseIdentifier: "repo")
+
+        let configureCell = {  (tableView: UITableView, row: Int, repository: Repository) -> UITableViewCell in
```

Comment:
You can stick to a single space after the `{`

Reply:
I miss Swiflint in this cases 😅